tests/Meilisearch.Tests/DocumentTests.cs

```diff
@@ -154,8 +154,8 @@ public async Task BasicDocumentsUpdate()
             var docs = await index.GetDocumentsAsync<Movie>();
             Assert.Equal("1", docs.First().Id);
             Assert.Equal("Ironman", docs.First().Name);
+            Assert.Null(docs.First().Genre);
```

Comment:
C# fields are by default null? In this case:

```c#
var movie = new Movie { Id = "1", Name = "Ironman" };

// is the same as 

var movie = new Movie { Id = "1", Name = "Ironman", Genre = null };
```

right?

Reply:
Well it depends of the kind of the field.
Reference type are null
Value type have a default value ie: int by default is 0